Formula/bitcoin.rb

```diff
@@ -25,13 +25,18 @@ class Bitcoin < Formula
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
-  depends_on "berkeley-db@5"
   depends_on "boost"
   depends_on "libevent"
   depends_on macos: :catalina
   depends_on "miniupnpc"
   depends_on "zeromq"
 
+  # berkeley db should be kept at version 4
+  # see the bitcoin build guides for more information:
+  # https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md
+  # https://github.com/bitcoin/bitcoin/blob/master/doc/build-unix.md
+  depends_on "berkeley-db@4"
```

Comment:
```suggestion
  depends_on "boost"
  # berkeley db should be kept at version 4:
  # https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md
  # https://github.com/bitcoin/bitcoin/blob/master/doc/build-unix.md
  depends_on "berkeley-db@4"
  depends_on "libevent"
  depends_on macos: :catalina
  depends_on "miniupnpc"
  depends_on "zeromq"
```
`brew audit` won't pass if dependencies are not listed in alphabetical order. (Squash this into your existing commit again please; thanks!)

Reply:
Done!